go/store/datas/pull.go

```diff
@@ -91,6 +92,18 @@ type TableFileEvent struct {
 	TableFiles []nbs.TableFile
 }
 
+func mapTableFiles(tblFiles []nbs.TableFile) ([]string, map[string]nbs.TableFile) {
```

Comment:
Comment describing the return type would be nice (table files keyed by ID)

Reply:
added